src/utils/1inch.js

```diff
@@ -80,8 +80,7 @@ export const getResponseData = async (url: string, errorMessage: string, toastMe
   return response.data;
 };
 
-export const parseAssets = (assets: Asset[]) => {
-  assets.forEach(asset => {
-    asset.code = asset.symbol;
-  });
-};
+export const parseAssets = (assets: Asset[]): any => assets.map((asset) => ({
```

Comment:
Returned type can be `Asset[]` instead of `any`, because the Asset model contains an optional `code` field as far as I remember